docs/rules/template-tag-spacing.md

```diff
@@ -0,0 +1,77 @@
+# Enforce Usage of Spacing in Tagged Template Literals (template-tag-spacing)
+
+(fixable) The `--fix` option on the [command line](../user-guide/command-line-interface#fix) automatically fixes problems reported by this rule.
+
+With ES6, it's possible to create functions called [tagged template literals](#further-reading) where the function parameters consist of a template literal's strings and expressions.
+
+This rule can force usage of spacing _between_ the tag function and the template literal.
```

Comment:
This section shouldn't be about the rule, it should just be about the concept the rule relates to. So, can you remove this line and do a bit more explanation around what a tagged template is where spacing is allowed? I'd like to see a complete example with the function definition.